modules/distribution/product/src/main/startup-scripts/profileSetup.sh

```diff
@@ -228,6 +256,8 @@ case $1 in
         ;;
 	-Dprofile=traffic-manager)
 		echo "Starting to optimize API Manager for the Traffic Manager profile"
+		replaceAxis2File
+		replaceRegistryXMLFile
 		disableTransportSenderWS
 		disableTransportSenderWSS
 		disableIndexingConfiguration
```

Comment:
When you replace the Axis 2 and registry file, you don't need to do the following. 

disableTransportSenderWS
disableTransportSenderWSS
disableIndexingConfiguration

To automate the traffic manager deployment, you only need to automate the step 1,2 and 4 in distributed guide [1]. 

[1] - https://docs.wso2.com/display/AM250/Deploying+WSO2+API-M+in+a+Distributed+Setup#DeployingWSO2API-MinaDistributedSetup-Step6.5-ConfigureandstarttheTrafficManager

Reply:
Removed those redundant steps "disableTransportSenderWS & disableTransportSenderWSS". By default, we decided to disable the registry indexing configuration.  We have already added the note as well in Product Profiles [1].

We didn't automate step 4 for in other cases as well. Actually, this script mainly built to automate the product profile configurations which discrbed ealier in doc [2]. 

[1] [https://docs.wso2.com/display/AM250/Product+Profiles](url)
[2] [https://docs.wso2.com/display/AM220/Product+Profiles#TM](url)